sparkle/src/components/ConversationMessageFeedbackSelector.tsx

```diff
@@ -23,6 +28,7 @@ export interface ConversationMessageFeedbackSelectorProps {
     }
   ) => Promise<void>;
   isSubmittingThumb: boolean;
+  getLastAuthor?: () => FeedbackAssistantBuilder;
```

Comment:
I may be lacking context but why not just `lastAuthor?: FeedbackAssistantBuilder` (without a function call) ?